controller/array_action/array_mediator_xiv.py

```diff
@@ -367,6 +368,23 @@ def delete_snapshot(self, snapshot_id):
 
         logger.info("Finished snapshot deletion. id : {0}".format(snapshot_id))
 
+    def _get_cli_host_by_name(self, host_name):
+        return self.client.cmd.host_list(host=host_name).as_single_element
+
+    def get_host_by_name(self, host_name):
+        cli_host = self._get_cli_host_by_name(host_name)
+        if not cli_host:
+            return None
+        fc_wwns = string_to_array(cli_host.fc_ports, ',')
+        iscsi_iqn = string_to_array(cli_host.iscsi_ports, ',')
+        connectivity_types = []
+        if fc_wwns:
+            connectivity_types.append(FC_CONNECTIVITY_TYPE)
+        if iscsi_iqn:
+            connectivity_types.append(ISCSI_CONNECTIVITY_TYPE)
+        initiators = Initiators(nvme_nqns=[], fc_wwns=fc_wwns, iscsi_iqns=iscsi_iqn)
+        return Host(name=cli_host.name, connectivity_types=connectivity_types, initiators=initiators)
```

Comment:
could we simply the call for `Host` init to something like:
```
Host(name='...', fc_wwns=fc_wwns, ...)
```
so the callers won't need to import and initialize `Initiators` separately, and the init would "calculate" the `connectivity_types` for them

Reply:
done

Reply:
I meant that the `Host` init caller won't have to specify the `connectivity_types`, and instead the init implementation would calculate them

Reply:
not every time we are using `connectivity_types` we also have the `host initiators`.

Reply:
good point